test/functional/p2p_getaddr_caching.py

```
@@ -4,7 +4,8 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test addr response caching"""
 
-from test_framework.messages import msg_getaddr
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)